plugins/generator-1.19.4/forge-1.19.4/workspacebase/run/server.properties

```diff
@@ -0,0 +1,35 @@
+generator-settings=
+op-permission-level=4
+allow-nether=true
+level-name=world
+enable-query=false
+allow-flight=false
+prevent-proxy-connections=false
+server-port=25463
+max-world-size=29999984
+level-type=DEFAULT
+enable-rcon=false
+level-seed=
+force-gamemode=false
+server-ip=
+network-compression-threshold=256
+max-build-height=256

```

Comment:
I just no5iced this, but since 11.8.x allows to build up to 320, we should also change the limit for the server.